sections/infrastructure.include

```diff
@@ -3316,7 +3316,7 @@ It covers
   the document.
 
   The <dfn export>rules for parsing a hash-name reference</dfn> to an element of type <var>type</var>,
-  given a context node <var>scope</var>, are as follows:
+  given a context node <var ignore=''>scope</var>, are as follows:
```

Comment:
since `scope` is not used in the algorithm, it makes more sense just to remove it, or check whether something is missing.

Reply:
Done.